<!-- Launcher for Kobuki's shooter controller -->

<launch>
  <include file="$(find kobuki_node)/launch/minimal.launch"/>
  <node pkg="nodelet" type="nodelet" name="kobuki_shooter_controller" args="load kobuki_shooter/ShooterControllerNodelet mobile_base_nodelet_manager">
    <param name="resources_path" value="$(find kobuki_shooter)/resources/"/>
<!--     <remap from="kobuki_shooter_controller/enable" to="mobile_base/enable"/> -->
<!--     <remap from="kobuki_shooter_controller/disable" to="mobile_base/disable"/> -->
<!--     <remap from="kobuki_shooter_controller/reset" to="mobile_base/reset"/> -->
    <remap from="kobuki_shooter_controller/events/bumper" to="mobile_base/events/bumper"/>
    <remap from="kobuki_shooter_controller/events/cliff" to="mobile_base/events/cliff"/>
    <remap from="kobuki_shooter_controller/events/wheel_drop" to="mobile_base/events/wheel_drop"/>
    <remap from="kobuki_shooter_controller/events/digital_input" to="mobile_base/events/digital_input"/>
    <remap from="kobuki_shooter_controller/commands/sound" to="mobile_base/commands/sound"/>
    <remap from="kobuki_shooter_controller/commands/led1" to="mobile_base/commands/led1"/>
    <remap from="kobuki_shooter_controller/commands/led2" to="mobile_base/commands/led2"/>
    <remap from="kobuki_shooter_controller/cmd_vel" to="mobile_base/commands/velocity"/>
  </node>
</launch>